content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -99,7 +99,7 @@ different Kubernetes components.
 | `DevicePlugins` | `true` | Beta | 1.10 | |
 | `DisableAcceleratorUsageMetrics` | `false` | Alpha | 1.19 | 1.19 |
 | `DisableAcceleratorUsageMetrics` | `true` | Beta | 1.20 | |
-| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | |
+| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
```

Comment:
It'd be like this:
```suggestion
| `DownwardAPIHugePages` | `false` | Alpha | 1.20 | 1.20 |
| `DownwardAPIHugePages` | `false` | Beta | 1.21 | |
```
(feel free to commit this via GitHub, fetch, and squash - or fix locally. Either is fine.)